Data/Primitive.hs

```diff
@@ -8,16 +9,16 @@
 -- Maintainer  : Roman Leshchinskiy <rl@cse.unsw.edu.au>
 -- Portability : non-portable
 --
--- Reexports all primitive operations
---
+-- Reexports all primitive operations.
+
 module Data.Primitive (
   -- * Re-exports
-  module Data.Primitive.Types
-  ,module Data.Primitive.Array
-  ,module Data.Primitive.ByteArray
-  ,module Data.Primitive.SmallArray
-  ,module Data.Primitive.PrimArray
-  ,module Data.Primitive.MutVar
+  module Data.Primitive.Types,
+  module Data.Primitive.Array,
+  module Data.Primitive.ByteArray,
+  module Data.Primitive.SmallArray,
+  module Data.Primitive.PrimArray,
+  module Data.Primitive.MutVar
```

Comment:
Why????

Reply:
I changed this to make the style more consistent with the other modules that use this (`Data.Primitive.Array`, `Data.Primitive.ByteArray`, `Data.Primitive.Ptr`, ...). This is a really minor change though, I can just revert it.

Reply:
I personally like commas first; didn't realize the other modules put them at the end. Spaces after commas for sure.

Reply:
I changed to the standard style of using leading commas.